glue/viewers/common/qt/data_viewer.py

```diff
@@ -53,6 +53,8 @@ class DataViewer(ViewerBase, QtWidgets.QMainWindow):
     LABEL = 'Override this'
 
     _toolbar_cls = None
+    # This defines the mouse mode to be used when no toolbar modes are active
+    _default_mouse_mode = None
```

Comment:
If this is intended to be a class, add a ``_cls`` suffix (and same inside the Toolbar class in places where it should be a class not an instance)